recipes-qt/qt5/qtbase_git.bb

```diff
@@ -33,6 +33,7 @@ SRC_URI += "\
     file://0015-double-conversion-support-AARCH64EB-and-arm-BE.patch \
     file://0016-Disable-ltcg-for-host_build.patch \
     file://0017-Qt5GuiConfigExtras.cmake.in-cope-with-variable-path-.patch \
+    file://0020-corelib-Include-sys-types.h-for-uint32_t.patch \
```

Comment:
What about nativesdk-qtbase? I'll add it there when cherry-picking to master-next and thud-next.

Reply:
yes we can add it to native as well. this change is general across qtbase